mkdocs.yml

```diff
@@ -85,6 +85,7 @@ nav:
         - Overview: user-guide/concepts/tools/tools_overview.md
         - Python: user-guide/concepts/tools/python-tools.md
         - Model Context Protocol (MCP): user-guide/concepts/tools/mcp-tools.md
+        - Executors: user-guide/concepts/tools/executors.md
```

Comment:
This is good if folks already know what executors are, but if they don't I'm not sure they would look here.  Is there a way we can make it more obvious when folks are reading about tools.  E.g. can we add a section to the tools_overview.md that actually calls out parallel vs sequential?